src/services/lomakkeet/perustelut/koulutukset.js

```diff
@@ -673,27 +702,19 @@ const johtaja = (orderCode, isReadOnly) => {
         anchor: "liitteet",
         styleClasses: ["pl-6 pt-6"],
         title: "Liitteet",
-        categories: [
+        components: [
           {
             anchor: "info",
-            components: [
-              {
-                anchor: "A",
-                name: "StatusTextRow",
-                properties: {
-                  title: HAKEMUS_OTSIKOT.LIITE_OHJE.FI
-                }
-              }
-            ]
+            name: "StatusTextRow",
+            styleClasses: ["w-full"],
+            properties: {
+              title: HAKEMUS_OTSIKOT.LIITE_OHJE.FI
```

Comment:
Let's fetch the localizations from our i18n based solution.

Reply:
Need to done separative, injectIntl has problems with this structure also.